reference-sheets/ControlTheory.md

```diff
@@ -0,0 +1,30 @@
+# Introduction
+Control theory is a widely-used concept in engineering that helps influence the behavior of a system to achieve a desired outcome. By applying methods like feedback and feedforward, we can correct errors and make systems more accurate and reliable. This guide covers the basics of these methods but primarily provides resources with more in-depth explanations of each topic. Many of the projects you will work on will implement these concepts, so it's important to understand how they function.
+
+To get started, read [this](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/introduction/control-system-basics.html) document on control systems. It shows how control systems are used in everyday life and why they’re important. It also breaks down basic concepts and uses diagrams to explain how most systems work.
+
+
+### Open vs. closed loop control
+
+**Open-loop control**, commonly known as `feedforward`, functions by sending control signals based on predetermined estimates, without adjusting for the system's actual state. This means it operates without feedback, relying only on initial assumptions about how the mechanism should behave. As seen below, feedforward is usually depicted as a straight path regardless of the "loop" in its name.
```

Comment:
There should be a discussion on _both_ types of feedforward (plant inversion and unmodeled dynamics, see page 11 of [Controls Engineering in FRC](https://file.tavsys.net/control/controls-engineering-in-frc.pdf)

Doesn't have to be in-depth at all, can link back to WPILib docs or even the textbook itself

Reply:
We should also add a resources doc for programming somewhere